src/Stripe.net/Stripe.net.csproj

```
@@ -5,7 +5,7 @@
     <Version>41.7.0</Version>
     <LangVersion>8</LangVersion>
     <Authors>Stripe, Jayme Davis</Authors>
-    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;net7.0;netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```
We can skip targeting net7.0 in the main project as we don't use any specific features from it. Targeting in the test project is still required.